internal/pkg/template/env.go

```diff
@@ -18,35 +18,38 @@ const (
 
 // Available env-controller managed feature names.
 const (
-	ALBFeatureName         = "ALBWorkloads"
-	EFSFeatureName         = "EFSWorkloads"
-	NATFeatureName         = "NATWorkloads"
-	InternalALBFeatureName = "InternalALBWorkloads"
-	AliasesFeatureName     = "Aliases"
+	ALBFeatureName                     = "ALBWorkloads"
+	EFSFeatureName                     = "EFSWorkloads"
+	NATFeatureName                     = "NATWorkloads"
+	InternalALBFeatureName             = "InternalALBWorkloads"
+	AliasesFeatureName                 = "Aliases"
+	AppRunnerPrivateServiceFeatureName = "AppRunnerPrivateWorkloads"
 )
 
 // LastForceDeployIDOutputName is the logical ID of the deployment controller output.
 const LastForceDeployIDOutputName = "LastForceDeployID"
 
 var friendlyEnvFeatureName = map[string]string{
-	ALBFeatureName:         "ALB",
-	EFSFeatureName:         "EFS",
-	NATFeatureName:         "NAT Gateway",
-	InternalALBFeatureName: "Internal ALB",
-	AliasesFeatureName:     "Aliases",
+	ALBFeatureName:                     "ALB",
+	EFSFeatureName:                     "EFS",
+	NATFeatureName:                     "NAT Gateway",
+	InternalALBFeatureName:             "Internal ALB",
+	AliasesFeatureName:                 "Aliases",
+	AppRunnerPrivateServiceFeatureName: "App Runner Private Services",
 }
 
 var leastVersionForFeature = map[string]string{
-	ALBFeatureName:         "v1.0.0",
-	EFSFeatureName:         "v1.3.0",
-	NATFeatureName:         "v1.3.0",
-	InternalALBFeatureName: "v1.10.0",
-	AliasesFeatureName:     "v1.4.0",
+	ALBFeatureName:                     "v1.0.0",
+	EFSFeatureName:                     "v1.3.0",
+	NATFeatureName:                     "v1.3.0",
+	InternalALBFeatureName:             "v1.10.0",
+	AliasesFeatureName:                 "v1.4.0",
+	AppRunnerPrivateServiceFeatureName: "v1.23.0",
```

Comment:
Should we bump the `deploy.LatestEnvTemplateVersion` to `v1.13`? I honestly can't remember why the environment template can't just be equal to the CLI version 🤔 maybe @Lou1415926 remembers. That would get rid of one more place we have to update when we make env template changes.

Reply:
I think env template version was first introduced because of `env upgrade`. 

Retroactively thinking, I assume the concern was that a major change in env template != a major change in Copilot, hence the versioning wasn't kept in sync.